WinPort/src/APITime.cpp

```diff
@@ -1,3 +1,6 @@
+#ifdef __HAIKU__
+#define _DEFAULT_SOURCE 1
```

Comment:
Suggest to move it to root CMakeLists.txt somehere here: https://github.com/elfmz/far2l/blob/master/CMakeLists.txt#L164 unless there're some contradictions..